gradle-plugin/src/main/groovy/akka/grpc/gradle/AkkaGrpcPlugin.groovy

```diff
@@ -1,20 +1,17 @@
 package akka.grpc.gradle
 
-
-import com.google.protobuf.gradle.Utils
+import com.google.protobuf.gradle.ProtobufPlugin
 import org.apache.commons.lang.SystemUtils
 import org.gradle.api.GradleException
 import org.gradle.api.Plugin
 import org.gradle.api.Project
-import org.gradle.api.artifacts.DependencyResolutionListener
-import org.gradle.api.artifacts.ResolvableDependencies
+import org.gradle.util.GradleVersion
+import org.gradle.util.VersionNumber
```

Comment:
We're now building using gradle 6.x. Is this change compatible with `5.6`? Later in the code we check the version is _at least_ `5.6` so we should support that.